config/jobs/kubernetes-sigs/descheduler/descheduler-presubmits-master.yaml

```diff
@@ -83,14 +83,14 @@ presubmits:
           requests:
             cpu: 6
             memory: 4Gi
-  - name: pull-descheduler-test-e2e-k8s-master-1-30
+  - name: pull-descheduler-test-e2e-k8s-master-1-31
     cluster: eks-prow-build-cluster
     annotations:
       testgrid-dashboards: sig-scheduling
-      testgrid-tab-name: pull-descheduler-test-e2e-k8s-master-1.30
+      testgrid-tab-name: pull-descheduler-test-e2e-k8s-master-1.31
     decorate: true
     decoration_config:
-      timeout: 30m
+      timeout: 20m
```

Comment:
Can you keep 30m? So https://github.com/kubernetes-sigs/descheduler/pull/1466 can pass the tests. It adds new e2e which in total with already existing takes more than 20 minutes to run.